website/src/components/Microbench/Microbench.jsx

```diff
@@ -0,0 +1,93 @@
+/*
+Copyright 2023 The Vitess Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React, {useState} from 'react';
+
+import '../Microbench/microbench.css'
+
+const Microbench = ({data, className, gitRefLeft, gitRefRight}) => {
+
+    const [maxHeight, setMaxHeight] = useState(70);
+
+    const handleClick = () => {
+        if (maxHeight === 70) {
+            setMaxHeight(400);
+          } else {
+            setMaxHeight(70);
+        }
+    };
+
+    return (
+        <div className={`microbench  ${className}`} style={{ maxHeight: `${maxHeight}px` }}>
+            <div className='space--between justify--content align--center'>
+                <span className='width--12em'>{data.PkgName}</span>
+                <span className='width--14em name'>{data.Name}</span>
+                <div className='width--18em space--between--flex hiddenMobile'>
+                    <span className='width--100'>{data.Right.Ops.toFixed(0)}</span>
+                    <span className='width--100'>{data.Left.Ops.toFixed(0)}</span>
```

Comment:
There is a small alignment issue with this component on large screens:
![image](https://github.com/vitessio/arewefastyet/assets/35779988/0396b455-0019-42ec-aeb0-1839b3731dc9)

The information that we see when clicking on `More` is not centered with the header.

Reply:
Looks good now